### ISSUE TITLE
We don't need to expose ssh port for Docker

### ISSUE THREAD

**Issue by kehao95** [tigergraph/ecosys#113]
https://github.com/tigergraph/ecosys/blob/master/demos/guru_scripts/docker/README.md

In this document you run the tiger-graph container as a server and exposed a SSH port mapped to 14022. 
Actually since we are using docker we can directly access the container shell via [docker exec](https://docs.docker.com/engine/reference/commandline/exec/) command.

```
sudo docker exec -it -u tigergraph tigergraph_dev bash
```

This will save the trouble configuring the ssh keys steps

**Comment by xpertmind** [comment]
True. My docker images (https://hub.docker.com/r/xpertmind/tigergraph/tags?page=1&ordering=last_updated) do not have SSH exposed. Note that we still need sshd internally, it is a part of the cluster installation and taking it out would couse only trouble.
